empiric-package/empiric/cli/contracts/oracle.py

```diff
@@ -62,6 +62,28 @@ async def deploy(
     return SUCCESS
 
 
+@app.command()
+@coro
+async def upgrade(cli_config=config.DEFAULT_CONFIG):
```

Comment:
maybe we should add some confirmation to avoid accidental invocation, like the `Y/n` with optional `-Y` arg with `apt-get`

Reply:
yeah we can add a guard